guvnor-bom/pom.xml

```diff
@@ -164,6 +164,12 @@
       </dependency>
       <dependency>
         <groupId>org.guvnor</groupId>
+        <artifactId>guvnor-ala-build-maven</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>tests</classifier>
```

Comment:
@porcelli I'll double check, but in the past, on other modules, I had to add the "tests" classifier in to be able to use e.g. some TestBase in guvnor, in other test class in kie-wb-commons. And this is basically the case here, I'm interested in using some test classes from guvnor in kie-wb-commons